packages/office-addin-lint/src/lint.ts

```diff
@@ -22,7 +22,7 @@ function getEsLintBaseCommand(): string {
 }
 
 export function getLintCheckCommand(files: string): string {
-  const eslintCommand: string = `${getEsLintBaseCommand()} ${files}`;
+  const eslintCommand: string = `${getEsLintBaseCommand()} ${files.replace(/ /g, '\\ ')}`; // Converting space to '\\'
   return eslintCommand;
```

Comment:
Can we create a common function to convert the file path ? Something like normalFilePath(filePath: string) : string